boards/arm/96b_wistrio/doc/96b_wistrio.rst

```diff
@@ -161,13 +151,12 @@ More detailed information on activating the ROM bootloader can be found in
 Chapter 29 of Application note `AN2606`_. The ROM bootloader supports flashing
 via UART, and I2C protocols.
 
-For flashing, `stm32flash`_ command line utility can be used. The following
-command will flash the ``zephyr.bin`` binary to the WisTrio board using UART
-and starts its execution:
-
-.. code-block:: console
+Here is an example for building and flashing the :ref:`hello_world` application.
```

Comment:
Sorry, I just edited my comment. So this should be modified as,

```
Here is an example for building and flashing the :ref:`hello_world` application using `stm32flash`_ command line utility:
```

Reply:
Done